### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by coke11**
Hey there! I updated the info site, but along with that came the following bugs
- Menu isn't working
- spacing wasn't working right for the unique description

I cleared all current translations and made one with google translate just to show it is possible. I will work on fixing these bugs really soon... Maybe one of you could help me?

ALSO - I am working on updating the formatting site. I have some good ideas :D

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,15 +21,19 @@
...
-        Dream up a character for this SDS!<br/>
-        Create a new character that could be used as a sprite in someones project. Your character could be anything; an orange dragon, a fluffy cloud, or even a bag of popcorn with a face!<br/>
-        Once you're done, give your character a personality! Write a short description about it, it can be anything; what it likes, hobbies, favorite color, etc.<br/>
-        Be sure to animate your character in some way to give it its own twist!
-        <br/>
-        <br/>
+        Celebrate the 100th SDS with us by creating a project with 10 squares!
+        <br>
+        <br>
+Create a project that contains 10 squares on the screen at a time, and 10 squares only. It could be a train driving on a track, with the train made out of 7 squares, and the tracks made out of 3. It could be an Ice-cream simulator with the ice cream made out of 5 squares, and the ice cream machine made out of the other 5 squares. The possibilities are endless!
```
Comment (TheGrits) [inline index.html:30]: Please indent this line so that it is even with the other lines, just easier for editing in the future

File: index.html [hunk 2]
```diff
@@ -108,8 +112,8 @@ <h2 id="curatorString">The SDS is curated by</h2>
           </li>
 
           <li>
-              <img src="//cdn.scratch.mit.edu/get_image/user/713038_170x170.png" alt="joletole Avatar" />
```
Comment (TheGrits) [inline index.html:111]: joletole is still a curator iirc (Thus please don't remove them)

File: index.html [hunk 3]
```diff
@@ -131,6 +135,21 @@ <h2 id="curatorString">The SDS is curated by</h2>
...
+              <img src="//cdn2.scratch.mit.edu/get_image/user/17384095_60x60.png" alt="Thinking_Upward Avatar" />
+              <span className="name">Thinking_Upward</span>
+          </li>
+          
+          <li>
+              <img src="//cdn2.scratch.mit.edu/get_image/user/7680480_60x60.png" alt="PackersRuleGoPack Avatar" />
+              <span className="name">PackersRuleGoPack</span>
+          </li>
+          
+          <li>
```
Comment (TheGrits) [inline index.html:149]: The list of curators should be alphabetical.

File: l10n.js [hunk 4]
```diff
@@ -1,74 +1,17 @@
 var l10n = {
   "en": {
-    "language":"English",
```
Comment (TheGrits) [inline l10n.js:3]: Uhh, why remove the perfectly fine English? You just need to change the 'unique' part of the description, not the whole thing.

File: l10n.js [hunk 5]
```diff
@@ -1,74 +1,17 @@
...
-    "curators":"The SDS is curated by"
-  },
-  "es": {
```
Comment (TheGrits) [inline l10n.js:9]: Whoa now! The 'standard' part of the translated descriptions are still very useful, we shouldn't be removing them! What we do is just change the 'unique' part to `Translation Needed!`

File: l10n.js [hunk 6]
```diff
@@ -1,74 +1,17 @@
...
+    "curators": "The SDS is curated by"
   },
   "fr": {
```
Comment (TheGrits) [inline l10n.js:9]: We'd prefer to keep hand translated translations over Google translated translations.

**Review by TheGrits — CHANGES_REQUESTED** [review]
Thanks for wanting to help out, but at this point this will do more damage than help. The HTML looks pretty good, just a few minor changes need to be made. The l10n.json file, however, is on the wrong track. We shouldn't be getting rid of translations just because the unique part of it is outdated, the standard part is still very useful! We should just be replacing the outdated uniques with "translation needed".

Thanks again for wanting to help!

**Comment by coke11** [comment]
Ok. So once I make the needed changes, this should be good to go?

File: l10n.js (hunk 6 above)
Comment (coke11) [reply l10n.js:9]: I know. This was just temporary.

File: l10n.js (hunk 4 above)
Comment (coke11) [reply l10n.js:3]: I fixed this later (in a nother commit)

File: l10n.js (hunk 5 above)
Comment (coke11) [reply l10n.js:9]: Ok. Then we would alert the translators on their profile?

File: index.html (hunk 2 above)
Comment (coke11) [reply index.html:111]: Oh, sorry about that. but doesn't it say for this studio?

File: index.html (hunk 3 above)
Comment (coke11) [reply index.html:149]: Oops. I will fix that asap

File: index.html (hunk 1 above)
Comment (coke11) [reply index.html:30]: Sorry about that

File: l10n.js (hunk 5 above)
Comment (TheGrits) [reply l10n.js:9]: @coke11 Yep, I'll contact some of the translators right now.

**Comment by TheGrits** [comment]
@coke11 I think so. Do you have any idea as to when you will be done?

**Comment by coke11** [comment]
@TheGrits Sorry - I have been really busy lately. I should have them done by Saturday :)

**Comment by TheGrits** [comment]
@coke11 No worries, I'm busy these days too!

**Comment by coke11** [comment]
@TheGrits Well, I am so busy these days XD! Why don't we just delete this and start fresh for the next one?

**Pull request closed without merging**